plugins/modules/elastic_endpoint_security.py

```diff
@@ -88,6 +88,8 @@ def main():
         verify_ssl_cert=dict(type='bool', default=True),
         agent_policy_id=dict(type='str'),
         agent_policy_name=dict(type='str'),
+        integration_title=dict(type='str', required=True),
+        integration_ver=dict(type='str', required=True),
```

Comment:
It was my understanding, this should require integration_title/version OR integration_name but all 3 are listed as required here.

Reply:
elastic_endpoint_security won't be used going forward. It can be deleted, but I wanted to keep it around just in case

Reply:
I have made the appropriate changes anyway